config/sync/views.view.su_opportunities.yml

```diff
@@ -562,6 +570,75 @@ display:
           limit: true
           error_message: true
           depth: 1
+        tid:
+          id: tid
+          table: taxonomy_index
+          field: tid
+          relationship: none
+          group_type: group
+          admin_label: ''
+          plugin_id: taxonomy_index_tid
+          operator: or
+          value:
+            781: 781
+            786: 786
+            791: 791
+            796: 796
+            801: 801
+            56: 56
+            811: 811
+            806: 806
```

Comment:
We can't wire up taxonomy terms in the view like this. Just change the operator to "any of".

Reply:
@pookmish Fixed.